### PR TITLE
wl-52624: Save 'regular' SDK files. +review WL-52131

### PR DESCRIPTION
**Pull request by DemienDM**

### CODE REVIEW

File: WellnessLiving/Wl/Profile/Edit/EditByTokenModel.php [hunk 1]
```diff
@@ -128,42 +153,56 @@ class EditByTokenModel extends WlModelAbstract
 
   /**
    * Exception class name.
-   * <tt>null</tt> if there was no mistake.
+   * `null` if there was no mistake.
    *
+   * @field class
```
Comment (Kasp42) [inline WellnessLiving/Wl/Profile/Edit/EditByTokenModel.php:158]: SDK does not support this tag.

File: WellnessLiving/Wl/Schedule/ScheduleList/StaffApp/ScheduleListModel.php [hunk 2]
```diff
@@ -178,10 +182,18 @@ class ScheduleListModel extends WlModelAbstract
    *     Count clients on waitlist.
    *   </dd>
    *   <dt>
+   *      int <var>id_option</var>
+   *    </dt>
+   *    <dd>
+   *      Appointment title display style.
+   *      Set only for appointments, for others it will be equal to 0.
+   *
```
Comment (Kasp42) [inline WellnessLiving/Wl/Schedule/ScheduleList/StaffApp/ScheduleListModel.php:190]: Missing link to Sid class

File: WellnessLiving/Wl/Staff/StaffElementModel.php [hunk 3]
```diff
@@ -40,27 +42,36 @@ class StaffElementModel extends WlModelAbstract
...
    *
    * @post post
    * @var int|null
    */
-  public $id_gender;
+  public $id_gender = null;
 
   /**
-   * ID of the default system role from {@link \WellnessLiving\RsPrivilegeRoleSid}.
+   * ID of the default system role.
```
Comment (Kasp42) [inline WellnessLiving/Wl/Staff/StaffElementModel.php:58]: Is it expected that this was deleted?

File: WellnessLiving/Wl/Video/VideoElementModel.php [hunk 4]
```diff
@@ -262,7 +263,7 @@ class VideoElementModel extends WlModelAbstract
    * If the file will be uploaded in parts, upload ID will be set.
    *
    * In this case, the file will not be uploaded to this API.
-   * Method {@link \WellnessLiving\Wl\Video\VideoElementModel::_fileVideoGet()} will not return the result.
+   * Method {@link VideoElementModel::_fileVideoGet()} will not return the result.
```
Comment (Kasp42) [inline WellnessLiving/Wl/Video/VideoElementModel.php:266]: Method does not exist in SDK

File: WellnessLiving/Wl/Video/VideoListModel.php [hunk 5]
```diff
@@ -62,48 +63,75 @@ class VideoListModel extends WlModelAbstract
...
+   * `null` if you need to return all the videos.
    *
    * @get get
-   * @var int
+   * @var int|null
+   */
+  public $i_page = null;
+
+  /**
+   * Sort order ID. `null` or 0 if order is undefined and depends on sort order.
```
Comment (Kasp42) [inline WellnessLiving/Wl/Video/VideoListModel.php:120]: Missing link to Sid class

File: WellnessLiving/Wl/Video/Watch/WatchModel.php [hunk 6]
```diff
@@ -30,7 +30,6 @@ class WatchModel extends WlModelAbstract
    * The source ID.
    *
    * @post post
-   * @see \WellnessLiving\Wl\Video\Watch\WatchSourceSid
```
Comment (Kasp42) [inline WellnessLiving/Wl/Video/Watch/WatchModel.php:33]: Is it expected that this was deleted?

**Review by Kasp42 — CHANGES_REQUESTED** [review]

File: WellnessLiving/Social/Facebook/Login/LoginModel.php [hunk 7]
```diff
@@ -13,20 +13,19 @@ class LoginModel extends WlModelAbstract
    * The application ID.
    *
    * This will be `null` if the application is being used to make the request.
-   * Use the application credential from the {@link \WellnessLiving\Core\Request\Model\Application\Credential\FacebookCredential} class for user authorization.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Social/Facebook/Login/LoginModel.php:16]: This class is not part of SDK and will not be added.

File: WellnessLiving/Core/Request/Api/Application/Credential/CredentialModel.php [hunk 8]
```diff
@@ -14,11 +14,11 @@ class CredentialModel extends WlModelAbstract
    * Accepted values: <ul>
    *   <li>
    *     <tt>972</tt> - The Google credential.
-   *     Use the {@link \WellnessLiving\Core\Request\Model\Application\Credential\GoogleCredentialData} object to send the credential.
+   *
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Core/Request/Api/Application/Credential/CredentialModel.php:17]: This class is not part of SDK and will not be added.

File: WellnessLiving/Social/Google/Plus/LoginModel.php [hunk 9]
```diff
@@ -13,20 +13,19 @@ class LoginModel extends WlModelAbstract
    * The application ID.
    *
    * This will be `null` for application use, which makes the request. Use the application credential from
-   * {@link \WellnessLiving\Core\Request\Model\Application\Credential\GoogleCredential} for user authorization.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Social/Google/Plus/LoginModel.php:16]: This class is not part of SDK and will not be added.

File: WellnessLiving/Wl/Appointment/Book/Finish/FinishMultipleModel.php [hunk 10]
```diff
@@ -213,8 +227,6 @@ class FinishMultipleModel extends WlModelAbstract
   /**
    * A list of payment sources to pay with.
    *
-   * The structure of this array corresponds with the structure of {@link RsPayForm::$a_pay_source}.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Wl/Appointment/Book/Finish/FinishMultipleModel.php:216]: This class is not part of SDK and will not be added.

File: WellnessLiving/Wl/Appointment/Book/Purchase/PurchaseModel.php [hunk 11]
```diff
@@ -467,7 +475,6 @@ class PurchaseModel extends WlModelAbstract
 
   /**
    * Image height in pixels. Please specify this value if you need image to be returned in specific size.
-   * In case this value is not specified returned image will have default thumbnail size {@link RsLoginLogo::THUMBNAIL_HEIGHT}.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Wl/Appointment/Book/Purchase/PurchaseModel.php:470]: This class is not part of SDK and will not be added.

File: WellnessLiving/Wl/Appointment/Book/Purchase/PurchaseModel.php [hunk 12]
```diff
@@ -476,7 +483,6 @@ class PurchaseModel extends WlModelAbstract
 
   /**
    * Image width in pixels. Please specify this value if you need image to be returned in specific size.
-   * In case this value is not specified returned image will have default thumbnail size {@link RsLoginLogo::THUMBNAIL_WIDTH}.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Wl/Appointment/Book/Purchase/PurchaseModel.php:479]: This class is not part of SDK and will not be added.

File: WellnessLiving/Wl/Book/Process/Quiz/QuizModel.php [hunk 13]
```diff
@@ -50,8 +54,8 @@ class QuizModel extends WlModelAbstract
   /**
    * The list of quiz response keys.
    *
-   * Keys refer to quiz keys, and values refer to response keys or special values from the
-   * {@link Wl\Quiz\Response\QuizResponse::RESPONSE_SKIP} constant.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Wl/Book/Process/Quiz/QuizModel.php:54]: This class is not part of SDK and will not be added.

File: WellnessLiving/Wl/Login/Promotion/PromotionPayPauseModel.php [hunk 14]
```diff
@@ -39,16 +39,16 @@ class PromotionPayPauseModel extends WlModelAbstract
...
    *
    * @get result
    * @var array[]|null
    */
-  public $a_pay_pause_list;
+  public $a_pay_pause_list = null;
 
   /**
-   * The end date of the current hold, in the local time zone. This can be set to a special value
-   * {@link Wl\Promotion\Pay\PromotionPayPause::DATE_END_INDEFINITE} to make the period indefinite until further action.
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Wl/Login/Promotion/PromotionPayPauseModel.php:51]: This class is not part of SDK and will not be added.

File: WellnessLiving/Social/Microsoft/LoginModel.php [hunk 15]
```diff
@@ -39,6 +39,8 @@ class LoginModel extends WlModelAbstract
    *
    * @post post
    * @var string
+   *
```
[the remark this answers is not in the document]
Comment (DemienDM) [reply WellnessLiving/Social/Microsoft/LoginModel.php:42]: Here and further.
It was decided to not clean the blank lines in the documentation that were created after deleting the sentences.

**Pull request merged**